lib/task/jobs/jobWorkerTask.class.php

```diff
@@ -128,6 +140,17 @@ function () use (&$counter) {
                 }
             }
         );
+
+        $jobsCompleted = $worker->getJobsCompleted();
+
+        // Force worker's __destruct() to run.
+        $worker = null;
```

Comment:
I prefer `unset($worker);` but I think it's entirely a stylistic choice.

Reply:
Huh, it looks like `unset($var)` removes `$var` from the symbol table too, which may be slightly slower, but frees slightly more memory? https://stackoverflow.com/questions/13667137/the-difference-between-unset-and-null

Probably too small a difference to really matter for either speed or memory utilization.

Reply:
Hmm... I had thought they were pretty equivalent, but it turns out there are differences.  Good idea - I will update to use unset().